src/main/java/sc/iview/commands/edit/NodeProperties.java

```diff
@@ -53,7 +53,6 @@
 
     @Override
     public void run() {
-        final NodePropertyEditor nodePropertyEditor = new NodePropertyEditor( sciView );
-        nodePropertyEditor.show();
+        sciView.toggleInspectorWindow();
```

Comment:
Yes, that makes sense. I have renamed it to Toggle Inspector and placed it in the View menu, in a new subsection within the View menu, because it didn't fit into any of the existing subsections.